### PR TITLE
Fix helm 3 crds installation + add some docs

### PR DESCRIPTION
**Pull request by PERES-Richard**
| Q               | A
| --------------- | ---
| Bug fix?        | [x]

### Why?
The actual installation with helm 3 use a crd-install hook wich is depreciated in helm 3. With this fix you can now install Casskop (CRD + operator) with just 1 line of helm.

### Additional context
It simply consist of copying CRDs from /deploy/crds to helm folders of Casskop & Multi Casskop.

### Checklist

- [x] Implementation tested
- [x] Logging code meets the guideline
- [x] User guide and development docs updated (if needed)
- [ ] Append changelog

### To Do
- [x] Test & changelog
- [ ] Deploy new artifact

### CODE REVIEW

File: website/docs/5_operations/3_5_backup_restore.md [hunk 1]
```diff
@@ -70,9 +71,9 @@ metadata:
   labels:
     app: cassandra
 spec:
-  backup:
+  cassandraBackup:
     name: nightly-cassandra-backup
```
Comment (cscetbon) [inline website/docs/5_operations/3_5_backup_restore.md:75]: no subkey

**Comment by cscetbon** [comment]
@PERES-Richard are you sure helm2 still works with those changes ? #133 seems to say it doesn't. And if doesn't I think we can drop support for helm 2.

**Review by cscetbon — CHANGES_REQUESTED** [review]

File: website/docs/5_operations/3_5_backup_restore.md (hunk 1 above)
Comment (PERES-Richard) [reply website/docs/5_operations/3_5_backup_restore.md]: done !

File: website/docs/2_setup/1_getting_started.md [hunk 2]
```diff
@@ -26,53 +26,22 @@ First, we need to create a `namespace` in order to host our operator & cluster (
...
-
-<Tabs
-  defaultValue="helm3"
-  values={[
-    { label: 'helm 3', value: 'helm3', },
-    { label: 'helm previous', value: 'helm', },
-  ]
-}>
-<TabItem value="helm3">
+Then install the chartusing Helm 3:
```
Comment (cscetbon) [inline website/docs/2_setup/1_getting_started.md:39]: chart` `using

**Review by cscetbon — APPROVED** [review]

File: website/docs/2_setup/1_getting_started.md (hunk 2 above)
Comment (cscetbon) [inline website/docs/2_setup/1_getting_started.md:39]:
```suggestion
Then install the chart using Helm 3:
```

**Review by fdehay — APPROVED** [review]
We should have left Kuttl out of the PR but it's ok

**Pull request merged**